src/app/components/modal/index.html

```diff
@@ -150,26 +150,15 @@
   </sky-demo-page-content>
 
 	<sky-demo-page-content sectionHeading="UX guidelines">
-		<div class="sky-subsection-heading">Modal title text</div>
-			<p>In general, use the following format for modal titles:</p>
-			<p><span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt; for &lt;indirect object&gt;</span></p>
-			<div style="margin-bottom: 20px;">
-				<img class="screenshot screenshot-info" src="~/assets/img/guidelines/wording/modal-with-object.png" alt="modal title: Edit address for Robert Hernandez"/>
-			</div>
-			<p>If there is not an indirect object or the user is the implied indirect object, you can exclude it:</p>
-			<p><span class="sky-emphasized">&lt;Verb&gt; &lt;direct object&gt;</span></p>
-			<div style="margin-bottom: 20px;">
-				<img class="screenshot screenshot-info" src="~/assets/img/guidelines/wording/modal-without-object.png" alt="modal title: Manage bank feeds"/>
-			</div>
-  	<div class="sky-subsection-heading">Usage</div>
-			<div class="sky-emphasized">Modal</div>
+		<div class="sky-section-heading">Usage</div>
+			<div class="sky-subsection-heading">Modal</div>
 			<p>A modal dialog is appropriate for forms that meet any of the following criteria:</p>
 			<ul>
 				<li>The task is functionally modal, and users must finish it before doing anything else.</li>
 				<li>Displaying the form inline would lead to substantial content shifting or require users to scroll to complete the form.</li>
 				<li>The form adds records from a paginated list or an infinite scroll list.</li>
```

Comment:
This third bullet is not reflected in the corresponding guidelines at https://developer.blackbaud.com/skyux/design/guidelines/form-design#form-containers. It seems like they should be the same. @Blackbaud-ToddRoberts, is there any reason to exclude this bullet from that corresponding list? If not, should we go ahead and add it as part of this PR?

Reply:
Also, should we link "paginated list" to the paging component and link "infinite scroll list" to the infinite scroll component?

Reply:
I don't think we need to link to paging/infinite scroll here, it's such ancillary info.

Can add the third bullet from modal to form design, was probably a copypasta error.

Reply:
Great. I'll add that third bullet to the other guidelines and push it up here.